libsolidity/analysis/TypeChecker.cpp

```diff
@@ -262,16 +268,20 @@ void TypeChecker::endVisit(InheritanceSpecifier const& _inheritance)
 			);
 		}
 		for (size_t i = 0; i < std::min(arguments->size(), parameterTypes.size()); ++i)
-			if (!type(*(*arguments)[i])->isImplicitlyConvertibleTo(*parameterTypes[i]))
-				m_errorReporter.typeError(
+		{
+			BoolResult result{false};
```

Comment:
Why not just assign it here from `type(...)->isImpl...` and then use it in the if?